### ISSUE TITLE
not get any notification when app closed or in the background

### ISSUE THREAD

**Issue by kenchu1178** [fechanique/cordova-plugin-fcm#251]
when app closed and in the background. push notification show in the bar. after taped it. app will show up and launch to start. but How can I get the notification in the app

**Comment by bendspoons** [comment]
See Payload Example in the Readme section. Add "click_action":"FCM_PLUGIN_ACTIVITY" when sending message. Solved my Problem with Notifications received while in background or closed.

**Comment by jonathan-chin** [comment]
@bendspoons I'm curious, the Payload Example also mentions a sound key, but I haven't been able to get it to work. Have you?

**Comment by kenchu1178** [comment]
THANKS, WORKING NOW

**Comment by bendspoons** [comment]
@jonathan-chin Yeah, sound works, but i didint do anything special for that. But be aware that you do not have to set the sound to true or sth like that, set it to "default" (as string)!

**Comment by jonathan-chin** [comment]
@bendspoons I'm using node-fcm and this is the payload I'm sending:

```
let message = {
		to: token,
		notification: {
		    tag: "new_messages",
		    title: `You have ${count} new message${count == 1 ? '' : 's'}`,
		    body: 'Click to read them!',
		    sound: 'default',
		    //badge: "100"
		},
		data: {
		    type: "chat"
		},
		collapse_key: "new_messages"
	    };
```

the sound flag is there and set to default but my phone doesn't do anything.

do you get vibration too?

**Comment by koenfaro90** [comment]
Thanks! Should be pointed out a bit clearer in the docs I'd say, but works fine!
